Mage/src/main/java/mage/abilities/keyword/ProtectionAbility.java

```diff
@@ -75,7 +75,7 @@ public ProtectionAbility copy() {
 
     @Override
     public String getRule() {
-        return "protection from " + filter.getMessage() + (removeAuras ? "" : ". This effect doesn't remove Auras.");
+        return "protection from " + filter.getMessage() + (removeAuras ? "" : ". This effect doesn't remove Auras" + (removeEquipment ? "" : ". This effect doesn't remove Equipment"));
```

Comment:
If that's true, should removeEquipment actually be a property on ProtectionAbility? RemoveAuras and RemoveEquipment are declared the same everywhere else within that class, except for that one place.

Reply:
I think removeEquipment isn't a sensible parameter to even have, because it's only used by this card, and the text is not "doesn't remove Equipment" but "doesn't remove Auras and Equipment you control". The relevant feature is the controller, not the subtype.